frontend/pages/SoftwarePage/SoftwareVulnerabilityDetailsPage/SoftwareVulnSummary/SoftwareVulnSummary.tsx

```diff
@@ -61,7 +61,6 @@ const SoftwareVulnSummary = ({
                   <TooltipWrapper
                     tipContent="The worst case impact across different environments (CVSS base score). This data is reported by the National Vulnerability Database (NVD)."
                     // to match neighboring tooltip wrapper
-                    position="top-start"
```

Comment:
@rachaelshaw  qq: Are we changing these to be below as well? I thought the ticket only said for form fields? (see current behavior recording)

Current behavior:
https://github.com/fleetdm/fleet/assets/71795832/2943c017-c4bc-4857-9a55-150870ab88ff